src/Experience/experience.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import ExperienceCard from './experience-card';
+import './experience.css';
+
+export default class Experience extends React.Component{
+
+    render(){
+        let arts = <ArtsConnect />
+        let tech = <StanTech />
+        let techIntro = <StanTechTop />
+        let portfolio = <Portfolio />
+        let rent = <ArtStuff />
+        return(
+            <div className='container'>
+                <div className='card-wrapper'>
+                    <ExperienceCard
+                        logo={require('../assets/logo-white.png')}
+                        logoAlt='Artist Connect'
+                        image={require('../assets/startup-weekend.png')}
+                        title='1st Place Startup Weekend'
+                        content={arts}
+                    />
+                    <ExperienceCard 
+                        logo={require('../assets/stantech-logo.png')}
+                        logoAlt='StanTech'
+                        title='Complete Site Redesign'
+                        topContent={techIntro}
+                        content={tech}
+                    />
+                    <ExperienceCard 
+                        logo={require('../assets/ck-logo.png')}
+                        logoAlt='Caren Keyes original portfolio'
+                        title='Original Portfolio Site'
+                        image={require('../assets/portfolio.png')}
+                        logoAlt='original portfolio website'
+                        content={portfolio}
+                    />
+                    <ExperienceCard 
+                        logo={require('../assets/rentartstufflogo2.png')}
+                        logoAlt='Rent Art Stuff'
+                        title='Rent Art Stuff MVP'
+                        image={require('../assets/artstuff.png')}
+                        content={rent}
+                    />
+
+                </div>
+            </div>
+        )
+    }
+}
+
+const ArtsConnect = () => {
+    return(
+        <div className='arts-connect expanded-content'>
+            <div>
+                <p>
+                I had the opportunity to be the only developer on an amazing team or artists and entrepreneurs.                </p>
+                <p>
+                Over about two days, I built out a demo of our space and gear sharing platform for all kinds of artists.                 </p>
+                <div className='card-links'>
+                    <p><a href='https://www.reviewjournal.com/business/entrepreneurs/artist-connect-takes-first-place-at-startup-weekend-las-vegas-2018-1531226/' target='_blank'>NEWS ARTICLE</a></p>
```

Comment:
You can use CSS `text-transform: uppercase;` instead of making the text actually all uppercase. Doesn't matter much, but that separates the style and content nicely.